City-of-Helsinki/berth-reservations-admin
# VEN-502 | Fetch leases data on single customer page

## Description :sparkles:
Single customer page:
- Fetch the leases data from the backend 

## Issues :bug:

### Closes :no_good_woman:
- [VEN-502](https://helsinkisolutionoffice.atlassian.net/browse/VEN-502): Single customer page: Fetch leases data from the backend

### Related :handshake:

## Testing :alembic:

### Automated tests :gear:️

### Manual testing :construction_worker_man:
- From the sidebar, click on `Asiakkaat`
- Click on one of the customers' names
- It goes to a single customer page. scroll down to see `VENEPAIKAT` card, that shows the previous leases a customer had

Note: most of the customers have no previous leases atm, you can check [this particular](http://localhost:3000/customers/UHJvZmlsZU5vZGU6N2FhZmQ3YjYtZWM1OC00NmMxLWJlYWEtNTZiMTNmZjJjYjc5%7D) customer has one previous lease.

## Screenshots :camera_flash:
![image](https://user-images.githubusercontent.com/23040926/76840980-c1646800-6840-11ea-935e-82c26a47cccd.png)

## Additional notes :spiral_notepad:
